### PR TITLE
Update NLS translation, remove unwanted spaces, add missing alignment

### PR DESCRIPTION
**Pull request by pshipton**
Corrects https://github.com/eclipse-openj9/openj9/pull/20222

### CODE REVIEW

**Review by keithc-ca — APPROVED** [review]
I notice that the space before the period at the end of the message was removed from many files, but not all: At least it's a step in the right direction.

**Comment by keithc-ca** [comment]
Jenkins compile alinux jdk21

**Comment by pshipton** [comment]
> I notice that the space before the period at the end of the message was removed from many files, but not all: At least it's a step in the right direction.

I thought I checked them all. I can update anything I missed.

**Comment by pshipton** [comment]
jenkins line endings check

**Comment by pshipton** [comment]
jenkins line endings check

**Comment by pshipton** [comment]
Jenkins compile amac jdk21

File: runtime/nls/shrc/j9shr_ca.nls [hunk 1]
```diff
@@ -6919,7 +6919,7 @@ J9NLS_CC_OSPAGE_SIZE_MISMATCH_V1.system_action=The JVM continues, but with memor
 J9NLS_CC_OSPAGE_SIZE_MISMATCH_V1.user_response=No action required
 # END NON-TRANSLATABLE
 
-J9NLS_SHRC_CM_PRINTSTATS_PAGESIZEE=Mida de la p\u00e0gina de mem\u00f2ria = %zu (mida de la p\u00e0gina actual del sistema operatiu %zu)
+J9NLS_SHRC_CM_PRINTSTATS_PAGESIZEE=Mida de la p\u00e0gina de mem\u00f2ria        = %zu (mida de la p\u00e0gina actual del sistema operatiu %zu)
```
Comment (keithc-ca) [inline runtime/nls/shrc/j9shr_ca.nls:6922]: I think one more space is needed for alignment, or is one of those unicode characters the width of two?

Comment (pshipton) [reply runtime/nls/shrc/j9shr_ca.nls:6922]: I aligned it with the previous field J9NLS_SHRC_CM_PRINTSTATS_FEATURE in the same language.

```
Funció                              = %s
Mida de la pàgina de memòria        = %zu (mida de la pàgina actual del sistema operatiu %zu)
```

**Pull request merged**